src/main/java/org/apache/commons/lang3/ObjectUtils.java

```diff
@@ -275,6 +315,45 @@ public static boolean allNotNull(final Object... values) {
         return true;
     }
 
+    /**
+     * Checks if all values in the array are {@code nulls}.
+     *
+     * <p>
+     * If any value is not {@code null}, the array is {@code null} or the array is empty (contains no elements) then
+     * {@code false} is returned. If all elements in array are {@code null} {@code true} is returned.
+     * </p>
+     *
+     * <pre>
+     * ObjectUtils.allNull(*)                = false
+     * ObjectUtils.allNull(*, *)             = false
+     * ObjectUtils.allNull(null)             = true
+     * ObjectUtils.allNull(null, null)       = true
+     * ObjectUtils.allNull(null, *)          = false
+     * ObjectUtils.allNull(*, null)          = false
+     * ObjectUtils.allNull(*, *, null, *)    = false
+     * ObjectUtils.allNull()                 = false
+     * ObjectUtils.allNull((Object[]) null)  = false
+     * </pre>
+     *
+     * @param values  the values to test, may be {@code null} or empty
+     * @return {@code false} if there is at least one non {@code null} value in the array, the array is {@code null}
+     * or array contains no elements, {@code true} if all values in the array are {@code null}s .
+     * @since 3.9
+     */
+    public static boolean allNull(final Object... values) {
+        if (values == null || values.length == 0) {
+            return false;
+        }
+
+        for (final Object val : values) {
```

Comment:
This can be replaced by `return firstNonNull(values) == null`, although then you're doubling up on the null check.